src/main/resources/metadata/tezos.carthagenet.conf

```diff
@@ -574,6 +574,10 @@
         hash {
           visible: true
           data-type: "hash"
+          reference: {
+            entity: "operations"
+            key: "operation_group_hash"
+          }
```

Comment:
Shouldn't be the other way round? 
The reference is from operations, referring to the containing group, via the `operation_group_hash` field.

Reply:
operations is the table with the details.

Reply:
operations is the table with the details.